### PR TITLE
Issue 345 loki data

### PR DESCRIPTION
**Pull request by mortenjc**
### Issue reference / description

The branch you merge from should already reference an event-formation-unit github ticket number. You can add a descriptive title, but if an issue is referenced, you don't have to.

## Checklist for submitter

- [ ] Check for conflict with integration test
- [ ] Unit tests pass

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel.

### CODE REVIEW

**Comment by mortenjc** [comment]
1) Moved LoKI code away from LoKIBase.cpp and into LoKIInstrument.cpp to
prepare for easier unit testing.
2) Updated data format to the latest (assumed) agreed version.

File: src/modules/loki/LokiInstrument.cpp [hunk 1]
```diff
@@ -37,6 +37,7 @@ LokiInstrument::LokiInstrument(struct Counters & counters,
       XTRACE(INIT, ALW, "Using the identity 'calibration'");
       uint32_t MaxPixels = LokiConfiguration.getMaxPixel();
       uint32_t Straws = MaxPixels/LokiConfiguration.Resolution;
+      XTRACE(INIT, ALW, "Inst: Straws: %u, REsolution: %u", Straws, LokiConfiguration.Resolution);
```
Comment (MortenHilkerSkaaning) [inline src/modules/loki/LokiInstrument.cpp:40]: typo

File: src/modules/loki/LokiInstrument.cpp [hunk 2]
```diff
@@ -100,15 +103,74 @@ void LokiInstrument::dumpReadoutToFile(DataParser::ParsedData & Section,
...
   CurrentReadout.EventTimeHigh = Data.TimeHigh;
   CurrentReadout.EventTimeLow = Data.TimeLow;
+  CurrentReadout.DataSeqNum = Data.DataSeqNum;
   CurrentReadout.AmpA = Data.AmpA;
   CurrentReadout.AmpB = Data.AmpB;
   CurrentReadout.AmpC = Data.AmpC;
   CurrentReadout.AmpD = Data.AmpD;
   CurrentReadout.RingId = Section.RingId;
   CurrentReadout.FENId = Section.FENId;
-  CurrentReadout.FPGAId = Data.FPGAId;
```
Comment (MortenHilkerSkaaning) [inline src/modules/loki/LokiInstrument.cpp:109]: was this made superflous?

Comment (MortenHilkerSkaaning) [reply src/modules/loki/LokiInstrument.cpp:109]: (apparently that was part of the point of this PR)

**Review by MortenHilkerSkaaning — APPROVED** [review]

**Pull request merged**